src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -16,9 +16,12 @@ namespace Microsoft.DotNet.ProjectModel.Compilation
     public class LibraryExporter
     {
         private readonly string _configuration;
+        private readonly string _runtime;
         private readonly ProjectDescription _rootProject;
+        private readonly string _buildBasePath;
+        private readonly string _rootPath;
 
-        public LibraryExporter(ProjectDescription rootProject, LibraryManager manager, string configuration)
+        public LibraryExporter(ProjectDescription rootProject, LibraryManager manager, string configuration, string runtime, string buildBasePath, string rootPath)
```

Comment:
What is the difference between rootPath and buildBasePath. I am not sure these are great names. They pretty much mean the same thing, unless you read through the code. Can we come up with better names for these?